src/Command/Pull/PullCommandBase.php

```diff
@@ -47,6 +47,24 @@ abstract class PullCommandBase extends CommandBase {
    */
   protected $drushHasActiveDatabaseConnection;
 
+  /**
+   * @param $environment
+   * @param $database
+   * @param $backup_response
+   *
+   * @return string
+   */
+  public static function getBackupPath($environment, $database, $backup_response): string {
```

Comment:
Extracted this method so that it can be used in tests to get a consistent path across OSes.